src/components/Menu/Menu.js

```diff
@@ -163,14 +173,23 @@ Menu.propTypes = {
   isAuthenticated: PropTypes.bool.isRequired,
 
   /**
-   * List of menu items to display in dropdown with links
+   * url path for My Dashboard page
+   */
+  dashboardPath: PropTypes.string,
+
+  /**
+   * url path for Security Setting page
+   */
+  securityPath: PropTypes.string,
+
+  /**
+   * url path for Profile page
    */
-  menuList: PropTypes.arrayOf(
-    PropTypes.shape({
-      key: PropTypes.string,
-      value: PropTypes.string,
-      path: PropTypes.string,
-      component: PropTypes.elementType,
```

Comment:
@shawn320 this removes support `component`. My previous commit was to introduce it so that other teams can pass in a Next Link component instead of the regular `<a/>`. I guess in your code you should accept `component` and default would be the `react-router-dom` Link component.

Reply:
Updated